core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/MockBigQueryServiceFactory.scala

```diff
@@ -122,4 +122,14 @@ class MockGoogleBigQueryService(queryResponse: Either[Throwable, TableResult]) e
   }
 
   override def deleteDataset(datasetName: String): IO[Boolean] = IO.pure(true)
+
+  override def getTable(datasetName: String, tableName: String): IO[Option[Table]] = {
+    if(tableName.equals("gcp_billing_export_v1_table_does_not_exist")) IO.none
+    else IO.pure(Some(null))
```

Comment:
I'm not too happy about this, but the `Some(null)` is completely intentional here. Creating an instance of a `Table` or a `Dataset` (for the method below) for mock purposes is surprisingly annoying. Since we don't actually use these objects for anything (we basically just call these methods to test for existence), returning `Some(null)` seemed like a somewhat okay way to deal with it. It's also just test code, not production code, so I'm not too concerned.